patches/minecraft/net/minecraft/client/player/LocalPlayer.java.patch

```diff
@@ -37,6 +65,15 @@
              this.f_108617_.m_104955_(new ServerboundPlayerCommandPacket(this, ServerboundPlayerCommandPacket.Action.START_FALL_FLYING));
           }
        }
+@@ -697,7 +_,7 @@
+       this.f_108614_ = this.m_21255_();
+       if (this.m_20069_() && this.f_108618_.f_108573_ && this.m_6129_()) {
+          this.m_21208_();
+-      }
++      } else if (this.isInCustomFluid() && this.f_108618_.f_108573_ && this.m_6129_()) this.getTouchingCustomFluid().sink(this);
```

Comment:
The else if should just be on the next line so as to not have the patch of removing a bracket.